splot/_viz_esda_mpl.py

```diff
@@ -351,7 +351,7 @@ def plot_moran_simulation(moran, aspect_equal=True,
     # plot distribution
     shade = kwargs.pop('shade', True)
     color = kwargs.pop('color', splot_colors['moran_base'])
-    sbn.kdeplot(moran.sim, shade=shade, color=color, ax=ax, **kwargs)
```

Comment:
Is this backwards compatible? How far? Shall we update requirements?

Reply:
`fill` was added as to keyword to replace `shade` in [`seaborn==0.11.0`](https://github.com/mwaskom/seaborn/blob/57b28140aa5e5d583642ad37f84a36afb690665e/seaborn/distributions.py#L1591-L1611). I'll go ahead and update requirements, etc.